test/unicode.js

```diff
@@ -7,6 +7,15 @@ const path = require("path");
 
 const tr46 = require("../index.js");
 
+// UTS #46 version 13.0.0 includes the following broken tests.
+// They both include U+18C4E, which prior to 13.0.0 was treated as disallowed
+// but became valid in 13.0.0. However, the IdnaTestV2.txt file does not appear
+// to be updated for this change, and still considers them invalid.
```

Comment:
Reported to Unicode:

```markdown
## IdnaTestV2.txt issue

The IdnaTestV2.txt that came with UTS #46 Version 13.0.0 has a minor
error. ToUnicode for "xn--mbm8237g..xn--7-7hf" is marked as failure with
reason "V6", but this should not be the case. (This domain appears
_twice_ around line 3070.)

The cause appears to be the lack of update for version 13.0.0. The
problematic domain contains the code point U+18C4E. While this code
point is "disallowed" in Unicode 12.0.0's IdnaMappingTable.txt (line
6397), it is "valid" under Unicode 13.0.0's (line 6450).

(Note, however, a similar domain "xn--mbm8237g.xn--7-7hf1526p" is
correct in the file and should remain forbidden. This is since it
additionally has the code point U+FE12, which is "disallowed" in both
12.0.0 and 13.0.0.)
```